llama_index/chat_engine/types.py

```diff
@@ -164,13 +206,11 @@ def chat_repl(self) -> None:
         """Enter interactive chat REPL."""
         print("===== Entering Chat REPL =====")
         print('Type "exit" to exit.\n')
-
         self.reset()
         message = input("Human: ")
         while message != "exit":
             response = self.chat(message)
             print(f"Assistant: {response}\n")
-            message = input("Human: ")
```

Comment:
I think this breaks the chat loop?

Reply:
You're right. I missed that, my bad. Pushed fix.